resources/habitat.rb

```diff
@@ -0,0 +1,40 @@
+require_relative '../libraries/helpers'
+
+actions :build, :build_and_publish
```

Comment:
I don't think you need to enumerate all the actions, if so be sure to change `:build_and_publish` to `:publish`.